### DISCUSSION TITLE
Ability to search for multiple tags

### DISCUSSION THREAD

**Discussion by gardener0** [standardnotes/forum#2566] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
Two things have surprised me as I've started using Standard Notes:
1. CMD-F does not act as a keyboard shortcut for searching as it does on nearly every other app on my Mac. 
2. I haven't figured out how to search for notes that have multiple tags, say "call" and "doToday" so I can see all of the calls I intend to do today.

**Describe the solution you'd like**
1. Have an Edit -> Find menu item with a CMD-F shortcut to b consistent with nearly every other app 
2. The simplest approach I can think of is to have a syntax for searching, similar to Google or DuckDuckGo.  For example, when I'm in my "call" folder, I could search for either "tag:doToday" or "#doToday" 

**Describe alternatives you've considered**
1. The only alternative is to keep using my mouse to navigate to the search bar and train myself to stop trying to use CMD-F.  This is slow for me, and may be an accessibility issue for others.
2. The only alternative I've brainstormed so far is using the Folders extension and trying to create a filter.  My first attempt doesn't appear to work:
!["call today", "and", ["tags", "includes", ["title", "includes", "call"]], ["tags", "includes", ["title", "includes", "doToday"]]] 

**Additional context**
Nothing springs to mind.

**Comment by JaspalSuri** [comment]
Hi @gardener0, thanks for the feedback. For #2, it looks like you've seen [Mo's response](https://github.com/standardnotes/forum/issues/262#issuecomment-921045002). 🙂

As for your first suggestion, we currently do not provide support for searching for text within your note as it requires a custom integration for each editor. For what it's worth, the Markdown Minimist Editor is currently the only editor (provided by us) that supports it & supports finding and replacing text, and the desktop app will highlight results within notes that match queries entered into the search bar.

**Comment by Robert-Ernst** [comment]
Chiming in on the topic of keyboard shortcut for the search.

I'm currently planning to move away from Standard Notes because I still can't use basic shortcuts on the windows client.
I also filed some other simple QoL improvement here: https://github.com/standardnotes/forum/issues/1235

I use SN as my daily driver and using a mouse to click into the search field, click on the note, click into the note to edit or copy some text is bothersome if you do that hundreds of times per day.

@mobitar said he will look into it here: https://github.com/standardnotes/forum/issues/288#issuecomment-506744618

CTRL + F for sure is not working for me. I don't use any extensions. I have the newest version installed.

**Comment by myreli** [comment]
I've changed the title of this issue so we can focus on the **ability to search multiple tags**, while this other issue discuss the experimental keyboard shortcuts available: https://github.com/standardnotes/forum/discussions/2567
